integration/tests/cook/util.py

```diff
@@ -430,10 +430,33 @@ def scheduler_info(cook_url):
     assert resp.status_code == 200, response_info
     return resp.json()
 
+@functools.lru_cache()
+def docker_image_for_default_pool():
+    """What is the docker image for the default pool."""
+    cook_url = retrieve_cook_url()
+    settings_dict = settings(cook_url)
+    default_pool_name = default_submit_pool() or default_pool(cook_url)
+    if 'default-container-for-pool' not in settings_dict:
+        return None
+    if default_pool_name not in settings_dict['default-container-for-pool']:
+        return None
+    default_container = settings_dict['default-container-for-pool'][default_pool_name]
+    if default_container["type"] == "docker":
+        return default_container["docker"]["image"]
+    return None
 
-def docker_image():
+
+def docker_image_explicit():
+    """Was a docker image set explicitly via an environmental variable? Used to gate tests that depend on default image."""
     return os.getenv('COOK_TEST_DOCKER_IMAGE')
 
+@functools.lru_cache()
+def docker_image():
+    env_image = docker_image_explicit()
+    if env_image is not None:
+        return env_image
+    return docker_image_for_default_pool()
```

Comment:
I don't get why we want this to start returning `docker_image_for_default_pool()`

Reply:
Like other comment. Moot.